src/UI/Web/Startup.cs

```diff
@@ -201,7 +201,7 @@ public static IServiceCollection AddCustomAuthentication(this IServiceCollection
                 options.ResponseType = useLoadTest ? "code id_token token" : "code id_token";
                 options.SaveTokens = true;
                 options.GetClaimsFromUserInfoEndpoint = true;
-                options.RequireHttpsMetadata = true;
+                options.RequireHttpsMetadata = Convert.ToBoolean(configuration["RequireHttpsMetadata"] ?? "True");
```

Comment:
Convert.ToBoolean(configuration["RequireHttpsMetadata"] ?? "False");

The above would work

Reply:
Applied.